### DISCUSSION TITLE
Is there a way to send invite link to multiple emails?

### DISCUSSION THREAD

**Discussion by laotala828** [supabase/supabase#11453] [Questions]
I am implementing a feature that allows users to invite his/her friend to join a community. The experience will be that users can input three emails at a time and click the invite button. 

The back-end will use this function 
**await supabase.auth.admin.inviteUserByEmail** 
to send the email. But right now, the function only accepts one string, how can I send an invite email to three email addresses at a time? Thanks.

**Comment by sumon-devxhub** [comment]
Is there have any update regarding this issue?
